src/main/resources/log4j.properties

```diff
@@ -1,15 +0,0 @@
-log4j.rootLogger=INFO, stdout, file
```

Comment:
Do not remove this file

Reply:
I have't this file, 
and after 'git pull' it didn't appear